scripts/global_ens/stats/exevs_global_ens_wmo_grid2grid_stats.sh

```diff
@@ -38,7 +38,31 @@ verify_type=$3
 
 if  [ $ens = gefs ] ||  [ $ens = cmce ] ; then
   if [ $verify_type = upper ] ; then 
+
+   if [ $ens = gefs ] && [ ! -s ${COMIN_wmo}.${VDATE}/gefs/gfsanl.t00z.deg1.5.f000.grib2 ] ; then
+       export subject="GFS analysis data missing for WMO gefs verif"
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
+       echo "Warning: No GFS analysis available for ${VDATE}" > mailmsg
+       echo Missing file is ${COMIN_wmo}.${VDATE}/gefs/gfsanl.t00z.deg1.5.f000.grib2  >> mailmsg
+       echo "Job ID: $jobid" >> mailmsg
+       cat mailmsg | mail -s "$subject" $maillist
+       exit
+    fi
+
+    if [ $ens = cmce ] && [ ! -s ${COMIN_wmo}.${VDATE}/cmce/cmcanl.t00z.deg1.5.f000.grib2 ] ; then
+       export subject="CMC analysis data missing for WMO cmce verif"
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
@BinbinZhou-NOAA this line (54) needs to be removed now that the variable is exported from the ecf script.